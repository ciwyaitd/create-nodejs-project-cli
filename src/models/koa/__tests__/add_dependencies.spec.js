'use strict'

describe('models/koa/add_dependencies', () => {
  it('should math the snapshot', () => {
    const mock = {}

    jest.mock('../../package_json/get')
    const get = require('../../package_json/get')
    get.mockImplementation(() => mock)

    jest.mock('../../../utils/cmd')

    const addDependencies = require('../add_dependencies')
    addDependencies()

    expect(mock).toMatchSnapshot()
  })
})
