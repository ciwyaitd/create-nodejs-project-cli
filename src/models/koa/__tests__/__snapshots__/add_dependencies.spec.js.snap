// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`models/koa/add_dependencies should math the snapshot 1`] = `
Object {
  "dependencies": Object {
    "koa": "^2.5.2",
    "koa-bodyparser": "^4.2.1",
    "koa-router": "^7.4.0",
  },
}
`;
